packages/orchestration/test/supports.ts

```diff
@@ -30,13 +31,15 @@ export {
   makeFakeTransferBridge,
 } from '@agoric/vats/tools/fake-bridge.js';
 
-export const commonSetup = async (t: ExecutionContext<any>) => {
+// The common setup cannot support a durable zone because many of the fakes are not durable.
+// They were made before we had durable kinds (and thus don't take a zone or baggage).
+// To test durability in unit tests, test a particular entity with `relaxDurabilityRules: false`.
```

Comment:
```suggestion
// To test durability in unit tests, use `provideDurableZone` instead of `makeHeapZone`. It can be passed in as a second optional argument to `commontSetup`.
```

It wasn't clear where we could set `relaxDurabilityRules: false`. I found `annihilate` and `reincarnate` from `swingset-liveslots` searching elsewhere but that doesn't seem to be the approach here.

The suggestion is for the demonstration in PR body but maybe we want to document `annihilate()` 🤷 . Curious to learn.